projects/js-packages/publicize-components/src/components/form/styles.module.scss

```diff
@@ -39,7 +39,7 @@
 }
 
 .broken-connection-btn {
-	&:global(.components-button.is-link) {
+	&:global(.components-button.is-link.components-button.is-link) {
```

Comment:
This fixes this issue of specificity that changed

| Without this change | With this change |
|--------|--------|
| <img width="268" alt="Screenshot 2025-02-17 at 10 18 37 AM" src="https://github.com/user-attachments/assets/a22e62c6-61b1-45ed-866c-0005939fd4af" /> | <img width="282" alt="Screenshot 2025-02-17 at 10 19 18 AM" src="https://github.com/user-attachments/assets/3f5ead05-866e-4768-8a72-c443770f0cc8" /> |